File: mario.c
#include <cs50.h>
#include <stdio.h>

int main (void)
{
    int n;
    do 
    { 
        n = get_int("Height: ");
    
    }
    while (n < 1 || n > 8);
    for (int i = 0; i < n; i++)
    {
        int s = (n - 1) - i;
        int h = i + 1;
        
        for (int j = 0; j < s; j++)
        {
            printf (" ");
        }
        for (int k = 0; k < h; k++)
        {
            printf("#");
        }
       
        printf("\n");
    }
}

File: mario.py
from cs50 import get_int

while True:
    n = get_int("Height\n")
    if n >= 1 and n <= 8:
        for i in range(n):
            for j in range(n - i - 1):
                print(" ",end = "")
            for k in range(i + 1):
                print("#",end ="")
            print()
        break
